modules/task.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable linebreak-style */
+/* eslint-disable no-unused-vars */
+// eslint-disable-next-line linebreak-style
+class Task {
+  constructor(index, description, completed) {
+    this.index = index;
+    this.description = description;
+    this.completed = completed;
```

Comment:
- [ ] It would be [better ](https://www.geeksforgeeks.org/what-is-the-role-of-src-and-dist-folders-in-javascript-jquery/)if you move this `modules` folder into the `src` folder from the `root` directory. :+1: